src/modules/workspace/reset.ts

```diff
@@ -22,26 +19,29 @@ export default async (name: string, options) => {
 
   log.debug('Resetting workspace', workspace)
 
+  let promptAnswer
   if (!preConfirm) {
-    await promptWorkspaceReset(workspace, account)
+    promptAnswer = await promptWorkspaceReset(workspace, account)
```

Comment:
Maybe just:
```
const promptAnswer = await promptWorkspaceReset(workspace, account)
if (!promptAnswer) {
  return
}
```

Reply:
The the if below could be removed

Reply:
I tried it before, but lint didn't like it :(